azure-pipelines.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+- job: BuildWindowsWithSonarCloud
+  displayName: Build for Windows\CMake With SonarCloud
+  pool:
+    vmImage: 'windows-latest'
+  variables:
+    BUILD_WRAPPER_DOWNLOAD_URL: https://sonarcloud.io/static/cpp/build-wrapper-win-x86.zip
+  steps:
+  - task: SonarCloudPrepare@1
+    inputs:
+      SonarCloud: 'SonarCloud' # Name of the SonarCloud service connection you created
+      organization: 'sc-sq-c-family-examples'
+      scannerMode: 'CLI'
+      configMode: 'file'
+  - bash: |
```

Comment:
Nice !

Reply:
After a bit of consideration. Do you think that using bash in the examples for Windows is helpful? I mean if people are building on windows, the bash syntax may be alien to them. Maybe it be worthwhile to migrate that to powershell, or something else windows native.

Reply:
Yep, rewritten in Powershell